src/Modules/MotdModule/Interfaces/IMotdSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Config.Net;
+using EvoSC.Modules.Attributes;
+
+namespace EvoSC.Modules.Official.MotdModule.Interfaces;
+
+[Settings]
+public interface IMotdSettings
```

Comment:
Add a Description attribute to each of these options giving a short summary explaining them.